docs/guides/local-apt-repository.rst

```diff
@@ -36,7 +37,7 @@ an official APT server to the world.
 Popular options could be your Ansible controller inside of a container or a
 micro-size instance on the cloud depending on your requirements for availability.
 
-Configure a throw away build server
+Configuring a throw away build server
 -----------------------------------
```

Comment:
The underline needs to match the header, otherwise it won't be parsed correctly by Sphinx.